libheif/api/libheif/heif_text.h

```diff
@@ -124,6 +139,22 @@ heif_error heif_image_handle_add_text_item(heif_image_handle *image_handle,
 LIBHEIF_API
 void heif_text_item_release(heif_text_item* text_item);
 
+/**
+ * Add an extended language property to the text item.
+ *
+ * This adds an RFC 5346 (IETF BCP 47) extended language tag, such as "en-AU".
+ *
+ * @param text_item the text item to query
+ * @param language the language to add
+ * @param out_optional_propertyId Output parameter for the property ID of the language property.
+ *                                This parameter may be NULL if the info is not required.
+ * @return heif_error_ok on success, or an error value indicating the problem
+ */
+LIBHEIF_API
+heif_error heif_text_item_add_extended_language(heif_text_item* text_item,
```

Comment:
If there can only be one `elng`, we might call it `_set_` instead of `_add_`.

Reply:
I have made this change.